docs/source/running_mypy.rst

```diff
@@ -339,7 +339,7 @@ the former has type hints and the latter does not. We run
 :option:`mypy -m mycode.foo <mypy -m>` and mypy discovers that ``mycode.foo`` imports
 ``mycode.bar``.
 
-How do we want mypy to type check ``mycode.bar``? Mypy's behaviour here is
+How do we want mypy to type check ``mycode.bar``? Mypy's behavior here is
```

Comment:
I don't think we've standardized on British or American spelling, so let's just leave this alone.